File: schemas/pet.py
def petEntity(item) -> dict:
    return {
        'id': str(item['_id']),
        'owner': str(item['owner']),
        'name': item['name'],
        'type': item['type'],
        'gender': item['gender'],
        'race': item['race'],
        'weight': item['weight'],
        'image': item['image'],
        'birthday': item['birthday']
    }

def petsEntity(items) -> list:
    return [petEntity(item) for item in items]

File: src/models/pet.js
const mongoose = require('mongoose')
const { Schema } = mongoose

const PetSchema = new Schema({
  owner: { type: Schema.Types.ObjectId, ref: 'User', require: true },
  name: { type: String },
  type: { type: String },
  gender: { type: String },
  race: { type: String },
  weight: { type: Number },
  image: { type: String },
  birthday: { type: Date, default: Date.now }
})

module.exports = mongoose.model('Pet', PetSchema)